src/Installer/Elastic.Installer.Domain/Model/Elasticsearch/Locations/LocationsModel.cs

```diff
@@ -142,57 +132,26 @@ public void SetDefaultLocations()
 			this._refreshing = false;
 		}
 
-		public bool SamePathAs(string pathA, string pathB)
+		private bool SamePathAs(string pathA, string pathB)
 		{
-			if (!string.IsNullOrEmpty(pathA) && !string.IsNullOrEmpty(pathB))
-			{
-				var fullPathA = this.FileSystem.Path.GetFullPath(pathA).TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
-				var fullPathB = this.FileSystem.Path.GetFullPath(pathB).TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
-				return 0 == string.Compare(fullPathA, fullPathB, StringComparison.OrdinalIgnoreCase);
-			}
-
-			return false;
-		}
+			if (string.IsNullOrEmpty(pathA) || string.IsNullOrEmpty(pathB)) return false;
+			
+			var fullPathA = this.FileSystem.Path.GetFullPath(pathA).TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
+			var fullPathB = this.FileSystem.Path.GetFullPath(pathB).TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
+			return 0 == string.Compare(fullPathA, fullPathB, StringComparison.OrdinalIgnoreCase);
 
-		public void SetWritableLocationsToInstallDirectory(bool sameFolder)
-		{
-			if (!sameFolder) return;
-			this._refreshing = true;
-			this.DataDirectory = this.FileSystem.Path.Combine(this.InstallDir, Data);
-			this.ConfigDirectory = this.FileSystem.Path.Combine(this.InstallDir, Config);
-			this.LogsDirectory = this.FileSystem.Path.Combine(this.InstallDir, Logs);
-			this._refreshing = false;
 		}
 
-		// ReactiveUI conventions do not change
-		// ReSharper disable InconsistentNaming
-		// ReSharper disable ArrangeTypeMemberModifiers
-		readonly ObservableAsPropertyHelper<bool> configureAllLocations;
-		public bool ConfigureAllLocations => configureAllLocations.Value;
-
 		readonly ObservableAsPropertyHelper<string> elasticsearchLog;
 		public string ElasticsearchLog => elasticsearchLog.Value;
 
-		bool placeWriteableLocationsInSamePath;
-		[Argument(nameof(PlaceWritableLocationsInSamePath))]
```

Comment:
Need to update documentation in README and on elastic.co with this change

Reply:
This was undocumented in the `README` as far as i could tell :)